packages/@uppy/core/src/index.ts

```diff
@@ -35,3 +35,7 @@ export type {
   Meta,
   Body,
 } from '@uppy/utils/lib/UppyFile'
+
+export type { UploadStatus } from './uppyEventAdapter.js'
+
+export { createUppyEventAdapter } from './uppyEventAdapter.js'
```

Comment:
These shouldn't live in core but in `@uppy/components`, it's specific to that and internal-only package. At some point we want to have a finite state machine in core, but until then this doesn't belong here.

Reply:
agree. fixed!